cartography/intel/crowdstrike/spotlight.py

```diff
@@ -0,0 +1,142 @@
+import logging
+from typing import Dict
+from typing import List
+
+import neo4j
+from falconpy.oauth2 import OAuth2
+from falconpy.spotlight_vulnerabilities import Spotlight_Vulnerabilities
+
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+def sync_vulnerabilities(
+    neo4j_session: neo4j.Session,
+    update_tag: int,
+    authorization: OAuth2,
+) -> None:
+    client = Spotlight_Vulnerabilities(auth_object=authorization)
+    all_ids = get_spotlight_vulnerability_ids(client)
+    for ids in all_ids:
+        vulnerability_data = get_spotlight_vulnerabilities(client, ids)
+        load_vulnerability_data(neo4j_session, vulnerability_data, update_tag)
+
+
+def load_vulnerability_data(
+    neo4j_session: neo4j.Session, data: List[Dict], update_tag: int,
+) -> None:
+    """
+    Transform and load scan information
+    """
+    ingestion_cypher_query = """
+    UNWIND {Vulnerabilities} AS vuln
+        MERGE (v:SpotlightVulnerability{id: vuln.id})
+        ON CREATE SET v.aid = vuln.aid,
+            v.cid = vuln.cid,
+            v.firstseen = timestamp()
+        SET v.status = vuln.status,
+            v.created_timestamp = vuln.created_timestamp,
+            v.closed_timestamp = vuln.closed_timestamp,
+            v.updated_timestamp = vuln.updated_timestamp,
+            v.cve_id = vuln.cve_id,
+            v.host_info_local_ip = vuln.host_info_local_ip,
+            v.remediation_ids = vuln.remediation_ids,
+            v.app_product_name_version = vuln.app_product_name_version,
+            v.lastupdated = {update_tag}
+        WITH v
+        MATCH (h:CrowdstrikeHost{id: v.aid})
+        MERGE (h)-[hv:HAS_VULNERABILITY]->(v)
+        ON CREATE SET hv.firstseen = timestamp()
+        SET hv.lastupdated = {update_tag}
+    """
+    logger.info(f"Loading {len(data)} crowdstrike spotlight vulnerabilities.")
+    vulns = []
+    cves = []
+    for item in data:
```

Comment:
non-blocker: L57 to L79 could go into a separaate `_transform_vulnerability_data()` to help with maintainability.